alf/metrics/metrics.py

```diff
@@ -0,0 +1,232 @@
+# Copyright (c) 2020 Horizon Robotics. All Rights Reserved.
```

Comment:
Why not just combining `metrics.py` and `metric.py`?

Reply:
just doing it the same way as tf_agents.  Plus file becomes too long if we merge.